pkg/usagetracker/tracker_store_collector.go

```diff
@@ -0,0 +1,23 @@
+package usagetracker
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var _ prometheus.Collector = &trackerStore{}
+
+var trackerStoreMetricDesc = prometheus.NewDesc(
+	"cortex_usagetracker_active_series",
```

Comment:
For when we'll productionise it (if we will) we should standardize metric prefix. My code has `cortex_usage_tracker`.

Reply:
I'll push a commit now so we don't forget.